src/appendix/osx-install.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/sh
```

Comment:
Can you make this a `.md` file with the script in the triple-backticks so that it formats properly?